readme/download.md

```
@@ -1,29 +1,100 @@
-Your download of <span class="downloaded-filename">Joplin</span> is in progress. If your download didn't start, <a href="#" class="download-click-here">click here</a>.
-</div>
+<!-- ✦ ZED-STYLE README ✦ -->
 
-<div class="get-it-desktop">
+<!-- ========================= -->
 
-## Joplin App for Desktop
-  
-Access your notes on Windows, macOS or Linux.
+<p align="center">
+  <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/Logo.png" width="72" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)